src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -592,14 +598,9 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
             },
 
             textContent: {
-                condition: el => domUtils.isScriptElement(el) || domUtils.isStyleElement(el),
-
-                get: el => {
-                    if (domUtils.isScriptElement(el))
-                        return removeProcessingHeader(el.textContent);
+                condition: el => domUtils.isElementNode(el),
```

Comment:
What do you mean?
`textContent` allows to obtain text of element and its descendants. So look [at test](https://github.com/DevExpress/testcafe-hammerhead/pull/1080/files#diff-daf554b967111c3bac68868c68540b3eR479). We should process `textContent` for `head` element otherways test will fail.